src/registry_schemas/schemas/agm_location_change.json

```diff
@@ -0,0 +1,71 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/agm_location_change",
+    "type": "object",
+    "definitions": {},
+    "title": "AGM Location Change Information Schema",
+    "required": [
+        "agmLocationChange"
+    ],
+    "properties": {
+        "agmLocationChange": {
+            "type": "object",
+            "required": [
+                "year",
+                "newAgmLocation"
+            ],
+            "properties": {
+                "year": {
+                    "type": "string",
+                    "description": "Year of the AGM, Must be on or after incorporation year and cannot be future year."
+                },
+                "newAgmLocation": {
+                    "type": "object",
+                    "description": "New AGM Location, Must be in B.C.",
+                    "required": [
+                        "streetAddress",
+                        "addressCity",
+                        "addressCountry",
+                        "addressRegion",
+                        "postalCode"
+                    ],
+                    "properties": {
+                        "streetAddress": {
+                            "type": "string",
+                            "maxLength": 50
+                        },
+                        "streetAddressAdditional": {
+                            "type": [
+                                "string",
+                                "null"
+                            ],
+                            "maxLength": 50
+                        },
+                        "addressCity": {
+                            "type": "string",
+                            "maxLength": 40
+                        },
+                        "addressCountry": {
+                            "type": "string"
+                        },
+                        "addressRegion": {
+                            "type": "string",
```

Comment:
Can you add max length? refer: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/address.json#L31

Reply:
I added it